_sips/minutes/2017-05-08-sip-minutes.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: sips
+title: SIP Meeting Minutes - 8th May 2017
+
+partof: minutes
+---
+
+# Minutes
+
+The following agenda was distributed to attendees:
+
+|Topic|Reviewers| Accepted/Rejected |
+| --- | --- | --- |
+| [SIP-NN - comonadic-comprehensions](http://docs.scala-lang.org/sips/comonadic-comprehensions.html) | Shimi Bandiel | Rejected |
+| [SIP-33 - Match infix & prefix types to meet expression rules](http://docs.scala-lang.org/sips/make-types-behave-like-expressions.html)| Oron Port | Pending |
+|Scala Library Changes|Adriaan Moors| **WHAT DO I PUT HERE** |
```

Comment:
Please check the third item, it doesn't seem to be a SIP, what do we put it in the columns?

Reply:
@jvican could you chime in here? (I wasn't present for the meeting)

Reply:
I would put "Scala-dev proposal" and link to https://github.com/scala/scala-dev/issues/323.